web-client/src/views/FileDocument/FileDocumentReview.tsx

```diff
@@ -192,222 +195,239 @@ export const FileDocumentReview = connect(
 
     return (
       <>
-        <Focus>
-          <h1
-            className="heading-1 margin-bottom-0"
-            id="file-a-document-header"
-            tabIndex={-1}
-          >
-            Review Your Filing
-          </h1>
-        </Focus>
+        {fileDocumentHelper.isAutoGenerated ? (
+          <FileEntryOfAppearanceReview
+            documentTitle={form.documentTitle}
+            pdfPreviewUrl={pdfPreviewUrl}
+            showModal={showModal}
+            onBack={() => navigateBackSequence()}
+            onCancel={() => formCancelToggleCancelSequence()}
+            onSubmit={() => submitExternalDocumentSequence()}
+          />
+        ) : (
```

Comment:
This is really what was added. The rest of the changes in this file are the result of nesting what was there before inside the 'else' of this ternary.